packages/website/pages/tokens.js

```diff
@@ -74,14 +74,14 @@ const TokenRow = ({ token, index, copied, onCopy, deleting, onDelete }) => {
       </TableElement>
       <TableElement {...sharedArgs} breakAll={false}>
         <form onSubmit={e => { e.preventDefault(); onCopy(token) }}>
-          <Button type="submit" small className="w-28">
+          <Button type="submit" small className="w-24">
```

Comment:
This is smaller than before. I believe it was this wide to allow "Copied!" to fit. Can you confirm that "Copied!" fits within this new smaller size, and the same for "Deleting..." below?

In general, please can we have screenshots with UI fixes like this? 🙏

Reply:
Of course, I'll add screenshots or videos to next UI fixes :)

Here's how it looks like now:
![image](https://user-images.githubusercontent.com/24696635/132045164-8eb03e54-29ec-46d8-ae59-14a7ddfc1531.png)